package.json

```diff
@@ -49,20 +49,20 @@
     "eslint-plugin-prettier": "^3.1.2",
     "husky": "^3.0.5",
     "jsdoc": "^3.6.3",
-    "knex": "~0.20.8",
+    "knex": "~0.21.0",
     "lint-staged": "^9.2.5",
     "mocha": "^7.0.1",
     "mysql": "^2.18.1",
     "nyc": "^15.0.0",
-    "pg": "^7.18.1",
+    "pg": "^8.0.3",
```

Comment:
bumped driver to peerDependence versions expected by knex

Reply:
Looks good.